### ISSUE TITLE
Please add option to compare custom versions

### ISSUE THREAD

**Issue by philipp-sumo** [mozilla/missioncontrol#204]
it would be great to have the option to manually set the versions you want to compare in the crashing measurements.
a usecase of this would be that at the start of a rollout like 58.0 it may be more indicative to compare it with the throttled initial versions of 57.0 & 56.0 than the preceding dot-releases.

**Comment by wlach** [comment]
Yep, I was initially resistant to make this overly general but I've come around to the idea that we need some kind of more general interface for comparing arbitrary versions (and probably arbitrary dimensions as well).
